README.md

```diff
@@ -33,7 +33,7 @@ Contribute your own observability pack to the New Relic One catalog by following
       - `rabbitmq01.png`
       - `rabbitmq02.png`
 
-    This process is similar for all other entity directories. Also, if you don't want to create entities for a given type, you can simply delete the corresponding directory.
+    This process is similar for all other entity directories. Also, if you don't want to create entities for a given type, delete the corresponding directory.
```

Comment:
If a user copies `_template` and leaves the alerts directory as-is, they get build errors on import.